analytics_plugin.py

```diff
@@ -30,8 +30,7 @@
 airflow_webserver_base_url = configuration.get("webserver", "BASE_URL")
 
 
-@provide_session
-def tasks_report(session) -> Any:
+def tasks_report() -> Any:
     untrusted_start_date = request.args.get("startDate")  # 2022-08-01
     untrusted_end_date = request.args.get("endDate")  # 2022-08-30
     try:
```

Comment:
Does it make sense to have a separate check, one for the start date and one for the end date? That way the logged error includes that detail: which parameter didn't validate. Seems like cleaner code to me.